galsim/image.py

```diff
@@ -317,7 +312,7 @@ def __init__(self, *args, **kwargs):
             try:
                 ncol = int(ncol)
                 nrow = int(nrow)
-            except Exception:
+            except:
```

Comment:
I know this isn't actually new in this branch, but since you highlighted it by editing it...

It probably makes more sense to just get rid of this try/except block.  The custom TypeError here isn't giving any additional information beyond whatever the system would raise from `ncol = int(ncol)` failing.  So it's probably cleaner to just let the user see the original traceback.